packages/form-js-editor/test/spec/features/properties-panel/groups/ValidationGroup.spec.js

```diff
@@ -150,8 +150,37 @@ describe('ValidationGroup', function() {
       fireEvent.input(requiredSelect, { target: { value: 'phone' } });
 
       // then
-      expect(editFieldSpy).to.have.been.calledOnce;
+      expect(editFieldSpy).to.have.been.calledWith(field, [ 'validate' ], { validationType: 'phone' });
+      expect(field.validate.validationType).to.equal('phone');
+    });
+
+
+    it('should clear custom', function() {
```

Comment:
As @philippfromme you should always test the DO and UNDO case to ensure that you properly modify the model.